packages/ag-charts-community/src/chart/dom/theme.css

```diff
@@ -58,6 +61,7 @@
         var(--ag-charts-toolbar-foreground-color) 6%
     );
     --ag-charts-toolbar-gap: var(--ag-charts-size);
+    --ag-charts-toolbar-font-family: var(--ag-charts-font-family);
```

Comment:
I would remove this as a separate variable, and have a single place to control font-family across all charts. 

(I can't see a use case for having a different font-family for toolbars to the rest of charts).